planning/behavior_path_planner/src/utils/goal_planner/default_fixed_goal_planner.cpp

```diff
@@ -16,12 +16,23 @@
 
 #include "behavior_path_planner/utils/goal_planner/util.hpp"
 #include "behavior_path_planner/utils/path_utils.hpp"
+#include "lanelet2_core/LaneletMap.h"
 
 #include <lanelet2_extension/utility/query.hpp>
 #include <lanelet2_extension/utility/utilities.hpp>
+#include <tier4_autoware_utils/geometry/boost_geometry.hpp>
+
+#include "autoware_auto_planning_msgs/msg/path_with_lane_id.hpp"
+
+#include <boost/geometry.hpp>
+
+#include <lanelet2_core/geometry/Polygon.h>
 
 #include <memory>
 
+using Point2d = tier4_autoware_utils::Point2d;
```

Comment:
Please define the using declarations within a namespace to limit their scope.

Reply:
moved them in
namespace behavior_path_planner{}